packages/strapi-admin/admin/src/components/Roles/Permissions/utils/createDefaultCTFormFromLayout.js

```diff
@@ -131,14 +131,19 @@ const createDefaultCTFormFromLayout = (
     // The object has the following shape: { [ctUID]: { [actionId]: { [property]: { enabled: false } } } }
     const contentTypesActions = Object.keys(subjectLayouts).reduce((acc, currentCTUID) => {
       const { actionId, applyToProperties } = current;
+      const currentSubjectLayout = subjectLayouts[currentCTUID];
+      const properties = currentSubjectLayout.properties.map(({ value }) => value);
```

Comment:
Is `properties` always defined in currentSubjectLayout object?

Reply:
Yes only for the collection types. We have another utility for the plugins and settings.